crd-generator/src/test/java/io/strimzi/crdgenerator/ExampleCrd.java

```diff
@@ -54,10 +55,10 @@
         }
     )
 )
-@OneOf({@OneOf.Alternative(@OneOf.Alternative.Property("either")), @OneOf.Alternative(@OneOf.Alternative.Property("or")), @OneOf.Alternative({@OneOf.Alternative.Property("mapStringString"), @OneOf.Alternative.Property("mapStringObject")})})
+@OneOf({@OneOf.Alternative(@OneOf.Alternative.Property("either")), @OneOf.Alternative(@OneOf.Alternative.Property("or")), @OneOf.Alternative({@OneOf.Alternative.Property("mapStringString"), @OneOf.Alternative.Property("mapStringQuantity"), @OneOf.Alternative.Property("mapStringObject")})})
```

Comment:
Why do you tie it into the alternative fields? I do not understand the relationship of it.

Reply:
Removed, it from alternatives. Wasn't sure if it should be part of it.